package.json

```diff
@@ -15,8 +15,9 @@
     "prettier": "prettier --ignore-path .gitignore --write \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "start-jupyter": "mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py",
-    "test": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:run",
-    "test-debug": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open"
+    "test-integration": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:run",
```

Comment:
the wording here felt off so I did some googling and discovered that it is more common to call this `test:integration`

It may be worth calling this `test:integration` and the new unit tests `test:unit` and then have the `test` command call both

 A reference: https://stackoverflow.com/questions/47606101/what-is-colon-in-npm-script-names

Reply:
That looks good to me - should I also change it to `test:integration:debug`?